buildSrc/subprojects/parser/src/test/java/org/apache/calcite/test/DialectGenerateTest.java

```diff
@@ -168,31 +169,31 @@ private void assertFileProcessed(String testName) {
   }
 
   @Test public void processFileEmpty() {
-    assertFileProcessed("empty");
+    assertFileProcessed("empty", true);
```

Comment:
nit: Use comment (here and elsewhere) to indicate the name of the parameter (e.g. /*specifyFilePath=*/)